### ISSUE TITLE
Renaming results comparison window

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#282]
I would like to be able to rename the results-comparison window, similarly to renaming PIs.

**Comment by msevestre** [comment]
Can't you rename them from the chart window? (Chart property, there is a name field there)

**Comment by PavelBal** [comment]
Yes, it works - I was not aware of this. It would still be good to have the context-menu option.

**Comment by tobiasK2001** [comment]
Yes I would support that it would be good to have the context option as the user expect it from the other features. Thanks! Tobias
